### DISCUSSION TITLE
Exporting as HTML to a web browser

### DISCUSSION THREAD

**Discussion by bfeddes** [YetiForceCompany/YetiForceCRM#16911] [How to]
Is there a way, rather than exporting a Quote, Project, or Ticket to an XLS file or a PDF file, that we can export it as an HTML file which opens locally in a web browser?

**Comment by mariuszkrzaczkowski** [comment]
there is no ready mechanism, you can combine the textparser with displaying in the WYSIWYG field and then you can generate various
